docs/guides/advanced-use-case/exclude-include-files.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Exclude/Include Files | Stash
+description: A guide on how to exclude/include subset of files during backup/restore process.
+menu:
+  docs_{{ .version }}:
+    identifier: advance-use-case-exclude-include-files
+    name: Exclude/Include Files
+    parent: advance-use-case
+    weight: 12
+product_name: stash
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+# Exclude/Include Subset of Files During Backup/Restore Process
+This guide will show you how to exclude/include subset of files during backup/restore process.
+
+## Before You Begin
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+- Install `Stash` in your cluster following the steps [here](/docs/setup/README.md).
+- Install Stash `kubectl` plugin following the steps [here](https://stash.run/docs/latest/setup/install/kubectl_plugin/).
+- You should be familiar with the following `Stash` concepts:
+  - [BackupConfiguration](/docs/concepts/crds/backupconfiguration.md)
+  - [BackupSession](/docs/concepts/crds/backupsession.md)
+  - [RestoreSession](/docs/concepts/crds/restoresession.md)
+  - [Repository](/docs/concepts/crds/repository.md)
+
+To keep everything isolated, we are going to use a separate namespace called `demo` throughout this tutorial.
+
+```bash
+$ kubectl create ns demo
+namespace/demo created
+```
+> **Note:** YAML files used in this tutorial are stored in [docs/examples/guides/advanced-use-case/exclude-include-files](/docs/examples/guides/advanced-use-case/exclude-include-files) directory of [stashed/docs](https://github.com/stashed/docs) repository.
+
```

Comment:
- Prepare workload
- Prepare backend
- Full backup
- Filtering during backup
  - Backup subset of files
  - Exclude subset of files
 
- Filtering during restore
  - Restore subset of files
  - Exclude subset of files
- Cleanup